dataset/validation/annotations/annotations/image_stars (1).txt

```diff
@@ -0,0 +1,2 @@
+4 0.375 0.52 0.3648923838210081 0.38027941206841825
```

Comment:
@Holandsoest @Woutje  this whole folder can be deleted, right?

Reply:
@Woutje @Maucee,
Yea, I store the whole `dataset/training/*` and `dataset/validation/*` in the **`Teams/General/AI dataset`**  
I don't think you need this